internal/api/container/remove.go

```diff
@@ -0,0 +1,113 @@
+package container
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/projecteru2/barrel/internal/sock"
+	"github.com/projecteru2/barrel/internal/utils"
+	minions "github.com/projecteru2/minions/lib"
+	log "github.com/sirupsen/logrus"
+)
+
+var regexDeleteContainer *regexp.Regexp = regexp.MustCompile("/(.*?)/containers/(\\w+)(\\?.*)?")
+
+type ContainerDeleteHandler struct {
+	netUtil        utils.NetUtil
+	inspectHandler ContainerInspectHandler
+	dockerSocket   sock.DockerSocket
+	minionsClient  minions.Client
+}
+
+type ContainerDeleteRequest struct {
+	Version  string
+	IDOrName string
+}
+
+func NewContainerDeleteHandler(dockerSocket sock.DockerSocket, minionsClient minions.Client, netUtil utils.NetUtil) ContainerDeleteHandler {
+	return ContainerDeleteHandler{
+		netUtil:        netUtil,
+		dockerSocket:   dockerSocket,
+		minionsClient:  minionsClient,
+		inspectHandler: ContainerInspectHandler{netUtil: netUtil, dockerSocket: dockerSocket},
+	}
+}
+
+func (handler ContainerDeleteHandler) Handle(response http.ResponseWriter, request *http.Request) (handled bool) {
+	var (
+		containerDeleteRequest ContainerDeleteRequest
+		matched                bool
+	)
+	if containerDeleteRequest, matched = handler.match(request); !matched {
+		handled = false
```

Comment:
handled is implicit ```false```, this line could be removed?, and no need an extra. ```matched``` variable here
```
var containerDeleteRequest ContainerDeleteRequest
if containerDeleteRequest, handled = handler.match(request); !handled {
    return
}

```

Reply:
agree, will follow